src/iomusxml.cpp

```diff
@@ -591,6 +600,42 @@ bool MusicXmlInput::ReadMusicXml(pugi::xml_node root)
         measure->AddChild(iter->second);
     }
 
+    // manage endings stack
+    if (!m_endingStack.empty()) {
+        std::vector<std::pair<std::vector<Measure *>, musicxml::EndingInfo> >::iterator iter = m_endingStack.begin();
+        Object *section = iter->first.front()->GetParent();
+        for (; iter != m_endingStack.end(); ++iter) {
```

Comment:
A really minor thing: I usually always initialize `iter` in the loop and not above. Could you change them?

Reply:
You are right. Here, I wanted to get a pointer to `section`, but I just realize that this pointer is creater further above in that method.